alacritty_terminal/src/term/mod.rs

```diff
@@ -973,8 +979,11 @@ impl<T> Term<T> {
 
         let old_cols = self.cols();
         let old_lines = self.screen_lines();
-        let num_cols = max(size.cols(), Column(MIN_SIZE));
-        let num_lines = max(size.lines(), Line(MIN_SIZE));
+
+        debug_assert!(size.cols() >= Column(MIN_COLS));
+        debug_assert!(size.lines() >= Line(MIN_LINES));
```

Comment:
Why are these assertions necessary?

Reply:
I think we should make it clear (for both the computer and code readers) that here we have conditions that the arguments must meet.